internal/ingress/types_equals.go

```diff
@@ -269,9 +269,20 @@ func (s1 *Server) Equal(s2 *Server) bool {
 	if !(s1.SSLCert).Equal(s2.SSLCert) {
 		return false
 	}
-	if s1.Alias != s2.Alias {
-		return false
+
+	for _, a1 := range s1.Aliases {
+		found := false
+		for _, a2 := range s2.Aliases {
+			if a1 == a2 {
+				found = true
+				break
+			}
+		}
+		if !found {
+			return false
+		}
 	}
+
```

Comment:
You need to assert the length of `Aliases` too. Given that this is not caught by test, I think we are also missing a test case where s1.Alias is {a} and s2.Alias is {a,b} and we assert they are different.

Reply:
done